extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -481,6 +507,104 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     }
   }
 
+  /**
+   * Sends RTC request as specified by rtcConfig. Returns promise which
+   * resolves to the targeting informtation from the request response.
+   * If the publisher has specified that noCache == true, then we will only
```

Comment:
Whether or not the response was cached should be controlled by the end point via cache headers.  I think it makes more sense to have an option that disables state while revalidate behavior (e.g. staleRevalidateDisabled: true).  The behavior is that the first request has no special cache headers and if SWR is not disabled, we send a second request with no-cache option